docker/mesos-cloud.conf

```diff
@@ -0,0 +1,4 @@
+[mesos-cloud]
+  http-client-timeout = 5s
+  state-cache-ttl     = 20s
+    
```

Comment:
Yeah, there's some overlap between what km-local & docker-compose.yml do in regards to that config. If you run km-local outside of docker-compose you'll need a different IP in there.